setup.py

```diff
@@ -64,66 +60,6 @@ def get_torch_arch_list() -> Set[str]:
     return set(arch_list)
 
 
-# First, check the TORCH_CUDA_ARCH_LIST environment variable.
```

Comment:
Maybe we can put all of this code into a function? Then we can just skip it if `torch.version.hip` is set :)

If you have a different preference, let me know!